database/scripts/format_report.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+require_relative 'lib/report_formatter'
+require 'json'
+
+report_name = ARGV.shift
+
+report = JSON.load_file(report_name)
+
+report['urgent']['build_regressions'] = ReportFormatter::build_regressions(report['urgent']['build_regressions'])
+
+# These are not implemented yet!!!
+report['urgent']['test_regressions_consecutive'] = []
+report['urgent']['test_regressions_flaky'] = []
```

Comment:
I think this PR should wait until this is finished, otherwise could be misleading even for the internal team.

Reply:
Agreed